packages/react-aria-components/src/DropZone.tsx

```diff
@@ -95,6 +100,10 @@ function DropZone(props: DropZoneProps, ref: ForwardedRef<HTMLDivElement>) {
         data-focus-visible={isFocusVisible || undefined}
         data-drop-target={isDropTarget || undefined} >
         <VisuallyHidden>
+          {/* Added as a workaround for a Chrome + VO bug where it will not announce the aria label */}
+          <div id={dropzoneId} aria-hidden="true">
```

Comment:
Originally, I wanted to do something like `<div id={dropzoneId} aria-label={ariaLabel} aria-hidden="true></div>` but Chrome + VO wouldn't announce the label (though it seemed fine on other platforms). I created a [codesandbox](https://codesandbox.io/s/button-labels-investigation-lz5nwf?file=/src/App.js) trying various combinations to see how Chrome + VO was announcing it